From 92651fb7e8136d37e769ee348adb49f60fb8d701 Mon Sep 17 00:00:00 2001
From: Colin Huang <u8813345@gmail.com>
Date: Mon, 11 May 2026 17:35:58 +0800
Subject: [PATCH] ARM: dts: aspeed: anacapa: add JTAG CPLD TRST pin to SGPIO
 map

Add JTAG_CPLD_TRST_R_N to the sgpiom0 pin name table on
Facebook Anacapa BMC.

This exposes the CPLD JTAG TRST signal through SGPIO,
allowing proper JTAG reset control during debug.

LORE: https://lore.kernel.org/all/20260511-add-jtag-trst-pin-v1-1-b0be2f7b2da5@gmail.com/

Signed-off-by: Colin Huang <u8813345@gmail.com>
---
 arch/arm/boot/dts/aspeed/aspeed-bmc-facebook-anacapa-evt2.dts | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm/boot/dts/aspeed/aspeed-bmc-facebook-anacapa-evt2.dts b/arch/arm/boot/dts/aspeed/aspeed-bmc-facebook-anacapa-evt2.dts
index 92aa58735..65ad503d0 100644
--- a/arch/arm/boot/dts/aspeed/aspeed-bmc-facebook-anacapa-evt2.dts
+++ b/arch/arm/boot/dts/aspeed/aspeed-bmc-facebook-anacapa-evt2.dts
@@ -1031,7 +1031,7 @@ &sgpiom0 {
 	"LEAK_DETECT_RMC_N", "RTM_MUX_R",
 	"HDR_P0_NMI_BTN_BUF_R_N", "FPGA_JTAG_SCM_DBREQ_N",
 	"Channel6_leakage_present", "whdt_sel",
-	"", "",
+	"", "JTAG_CPLD_TRST_R_N",
 	"", "",
 	"", "",
 	"", "",
